### ISSUE TITLE
Should we delete this?

### ISSUE THREAD

**Issue by henrypinkard** [micro-manager/MM3rdparty#2]
@marktsuchida, this repo is not in use because the real version lives on svn, correct? Should we delete this to avoid confusion?

**Comment by marktsuchida** [comment]
Yes, agreed, it should be archived or deleted, since we have no plans to use a Git version of 3pp. Maybe archive just because it does contain some work -- although nothing that we are likely to be able to salvage.
